R/utils.R

```diff
@@ -47,3 +47,25 @@ is_string = function(x) {
 is_path = function(path) {
   is_string(path) && startsWith(path, "/")
 }
+
+to_http_date = function(dtm) {
+  if (is.null(dtm)) {
+    return(NULL)
+  }
+  old_loc = Sys.getlocale("LC_TIME")
```

Comment:
couple of comments about temporary changing locale would be helpful for review and for future reference. What is the purpose?

Reply:
In my system locale a day and month abbreviation do not works as expected. `stringi` or `lubridate` can use locale param. I don't know how force the English names.

Reply:
From the R man `?strptime`:

```r
## An RFC 5322 header (Eastern Canada, during DST)
## In a non-English locale the commented lines may be needed.
## prev <- Sys.getlocale("LC_TIME"); Sys.setlocale("LC_TIME", "C")
strptime("Tue, 23 Mar 2010 14:36:38 -0400", "%a, %d %b %Y %H:%M:%S %z")
## Sys.setlocale("LC_TIME", prev)
```

Reply:
could you please add this as comment to the code?

Reply:
Sure.